src/main/resources/l10n/JabRef_en.properties

```
@@ -2257,6 +2257,7 @@ Previous\ preview\ style=Previous preview style
 
 (\ Note\:\ Press\ return\ to\ commit\ changes\ in\ the\ table\!\ )=( Note\: Press return to commit changes in the table\! )
 Reset=Reset
+New\ inproceedings=New article in a conference proceedings
```

This needs to be exactly identical for the consistency test.
See: https://github.com/JabRef/jabref/runs/1271802665?check_suite_focus=true

```suggestion
New\ inproceedings=New inproceedings
```